### DISCUSSION TITLE
K-WASM 컴파일된 WASM 활용 예제 공개합니다.

### DISCUSSION THREAD

**Discussion by Hyeopgeon-Lee** [K-PaaS-incubator/k-wasm-app#7] [Announcements]
웹브라우저에서 C언어 함수를 5초마다 호출하는 간단한 예제입니다.

아래 URL에서 확인가능하며, 소스는 Git에 공개하였습니다.

https://sample.k-wasm.kr/html/example2/sample.html

**Comment by chlwnstkd** [comment]
좋은 기술 감사합니다
